radboudumc-theme/sass/_typography.scss

```diff
@@ -12,17 +12,39 @@ body {
   // No web fonts are used so maximize client rendering of the system font
   text-rendering: geometricPrecision;
   -webkit-font-smoothing: antialiased;
+
+  color: rgb(50, 50, 50);
 }
 
 // Default settings for headers
 h1,h2,h3,h4,h5,h6 {
   color: $primary;
 }
-h1,h2,h3 {
+
+h1,h2,h3,h4,h5,h6  {
+  font-weight: 300;
+}
+
+.block-home h1 {
+  font-weight: 300;
+}
+
+h1 {
+  font-size: 2rem;
+}
+
+// Sub-titles
+h2 {
+  font-size: 1.25rem;
   font-weight: 300;
+
+  margin-top: 1.5rem;
 }
-h4,h5,h6 {
-  font-weight: 400;
+
+main.content-home h2 {
```

Comment:
Does this need the tag (main) and the class? Can be simplified probably by using just the class.

Reply:
I will change it, I just always like to be as specific as possible with CSS to not get any unwanted side-effects